### ISSUE TITLE
Question: Calling 3rd-party CLI from oclif command.

### ISSUE THREAD

**Issue by Lighttree** [oclif/oclif#722]
Do you want to request a *feature* or report a *bug*?
-----------------------------------------------------
Question

Problem
-----------------------------
I would like to "wrap" another CLI by my `oclif` CLI which is some kind of "facade" for underlying implementation.

For example, I would like to call `rollup` CLI from my `oclif` CLI. The problem is that `rollup` doesn't provide JS-based API which is 100% equivalent to the `rollup` CLI command. There is quite a lot of logic in the CLI command itself.

So I was hoping to do something like:

```js
class BuildCommand extends Command {
    async run() {
        const ROLLUP_CONFIG_PATH = path.resolve(__dirname, '../configs/rollup.config.js')

        return new Promise((res, rej) => {
            exec(`NODE_ENV=production rollup -c ${ROLLUP_CONFIG_PATH}`, rej)
        })
    }
}
```

While this works it removes all prompts produced by `rollup`. But it would be nice to make my:

`$ myCustomCli build [flags]` command equivalent to `$ rollup [flags]` call.

Is this possible? Maybe I can find some examples?

P.S Maybe it makes sense to enable GitHub "Discussions" functionality for `oclif` repo? Because I didn't find a proper place where aks "question".

**Comment by henrik242** [comment]
> P.S Maybe it makes sense to enable GitHub "Discussions" functionality for oclif repo? Because I didn't find a proper place where aks "question".

I agree! Couldn't find a forum or slack to discuss with other oclif'ers.
